.babelrc

```diff
@@ -5,7 +5,7 @@
       {
         "useBuiltIns": true,
         "targets": {
-          "node": "4.3"
+          "node": "6.9.0"
```

Comment:
```
"node": "6"
```

Reply:
Where can I find this recommondation ?

Reply:
@michael-ciniawsky as i said above only `6.9` and above versions are lts, we should use `6.9` (6.9.0) here